bin/run-fisher.sh

```diff
@@ -14,4 +16,5 @@ python -u DeepSpeech.py \
   --epoch=20 \
   --display_step 1 \
   --validation_step 5 \
+  --checkpoint_dir "$XDG_DATA_HOME/$(basename ${0%.*})" \
```

Comment:
Hard code it is. I did not want to overly complicate the script, and if people auto generate scripts they already know the file name so it will just require adding one more variable. I tried, and at least I learned some stuff. =)

I will include this in my next commit.